CI-CD/AzureDevOps/azure-pipelines_cloud_native_container_scan.yml

```diff
@@ -0,0 +1,19 @@
+name: 'mend-cloud-native-cli-scan'
+
+trigger:
+- master
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- script: |
+    echo Downloading Mend CLI
+    sudo curl https://downloads.mend.io/production/unified/latest/linux_amd64/mend -o /usr/local/bin/mend && sudo chmod +x /usr/local/bin/mend
```

Comment:
sudo is not required as the command runs in a dockerized env

Reply:
fixed, and added UUID parameter.